src/core/include/openvino/pass/manager.hpp

```diff
@@ -20,11 +20,10 @@ namespace pass {
  */
 class OPENVINO_API Manager {
 public:
-    Manager();
```

Comment:
```cpp
    std::shared_ptr<PassConfig> m_pass_config{};
    std::vector<std::shared_ptr<PassBase>> m_pass_list{};
 ```
 Default init these members, then default ctor should initialize correctly all members with name "UnamedManager".
 
Or remove default initialization in hpp and initialize all members in constructors defined in cpp file.